serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/servicerecord/EDUServiceRecord.java

```diff
@@ -14,13 +14,13 @@
     private String serviceEditionCode;
 
     EDUServiceRecord(ServiceregistryProperties properties, String pemCertificate, String endpoint, String orgnr) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "properties". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1172)